README.md

```diff
@@ -34,3 +34,5 @@ The only parts that are not included in the server and client at this moment are
 - Run tests on both the server and client when new pushes are made to any branch in the project
 - Run linting on both the server and client when new pushes are made to any branch in the project
 - Deploy the client when a new push is made to the `main` branch 
+
+Note that in order to execute shell files (files with an `.sh` extension), then need to be ran in Github Actions with `sh shell_file_name.sh`. If they are ran with `. ./shell_file_name.sh` they will not run correctly.
```

Comment:
This is a little ambiguous. It seems to imply that I can't run the shell script unless I'm doing so through Github Actions (which shouldn't be true). Could you rephrase it a bit?

Reply:
You are right that is vague. Just updated it to the below:

Note that in order to execute `.sh` files inside of a workflow (the `.yaml` files in `.github/workflows`), they need to be ran with `sh shell_file_name.sh` INSTEAD of something like `. ./shell_file_name.sh`. If they are ran with `. ./shell_file_name.sh` they will not apply to the environment correctly. 

Does that make more sense or should it be a little more descriptive or changed? Trying to think of the right wording to make it better so suggestions are welcome.

Reply:
That makes more sense for sure! Thanks for the fix.